languages/cpp2/src/main/antlr4/de/jplag/cpp2/grammar/CPP14Parser.g4

```diff
@@ -790,6 +790,7 @@ theOperator:
 	| GreaterEqual
 	| PlusAssign
 	| MinusAssign
+	| DivAssign
```

Comment:
Maybe add a comment here that this was changed from the original grammar. You could also make a PR to the antlr repo.